_posts/2020-06-20-Oskari_3D_videos.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "3D capabilities of Oskari with videos"
+excerpt : "3D view and terrain height profile tool"
+date:   2020-06-20 13:00:00 +0300
+categories: [news]
+---
+
+# Testing Oskari 3D-possibilities 
+
+Oskari Joint Development Forum has created a video how to use 3D datasets.
+
+## Spin the world with the Oskari demo website
+
+If you have not tried yet, test Oskari 3D possibilities with the demo website. 
+
+[See the whole video about 3D in demo Oskari.](https://youtu.be/ISKkL1_-svc)
+
+1. First test the "Blue Marble" as background map and switch to 3D view [go to demo Oskari from here](https://demo.oskari.org/?zoomLevel=1&coord=431052.8380458826_5167185.112970014&mapLayers=8+100+,6+100+default&uuid=055af770-c777-4433-8127-28ce2d06c944&noSavedState=true&showIntro=false)
```

Comment:
Database on demo.oskari.org is wiped each night so linking with uuid will only work for a day or so.